pandas/core/groupby/ops.py

```diff
@@ -799,14 +799,37 @@ def apply(
         # This calls DataSplitter.__iter__
         zipped = zip(group_keys, splitter)
 
+        i = 0
         for key, group in zipped:
+            # BUG:47350 if replaced 1 by hamedgibago
+            # if key not in data.index and is_datetime64_any_dtype(data.index):
```

Comment:
Is this comments or old code?

Reply:
Yes, I have to remove this comment for my old code too.

Reply:
Looks like this old code is still here